src/Workspaces/SharedUtilitiesAndExtensions/Compiler/CSharp/CodeStyle/CSharpCodeStyleOptions.cs

```diff
@@ -7,10 +7,18 @@
 using System.Linq;
 using Microsoft.CodeAnalysis.AddImports;
 using Microsoft.CodeAnalysis.CodeStyle;
+
+#if CODE_STYLE
+using Microsoft.CodeAnalysis.Internal.Options;
+
+namespace Microsoft.CodeAnalysis.CSharp.Internal.CodeStyle
```

Comment:
this is super confusing to read and understand.  i also just am hating the ifdef shenanigans.